Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
+
+RUN mkdir /core
+COPY src/ /core/
+WORKDIR /core
+RUN apk add python3
+RUN pip3 install --upgrade pip
+RUN pip3 install -r requirements.txt
+RUN chmod +x ./*.py
+
```

Comment:
You will need to expose at leas tone port, see:
https://docs.docker.com/engine/reference/builder/
the option:
EXPOSE ...

Reply:
its already done, you probably only looked at the master code, but the latest is the one on the "socket" branch...this PR is actually obsolete, I'll change it when I get a chance